### ISSUE TITLE
💅 noDuplicateProperties falst positive with postcss/mantine

### ISSUE THREAD

**Issue by astanciu** [biomejs/biome#6615]
MantineUI lib uses this postcss plugin:
https://github.com/mantinedev/postcss-preset-mantine

Which allows css code like this:

```css
.class {
  color: red;
  
  @mixin hover {
    color: blue;
  }
}
```

And this triggers the `noDuplicateProperties` rule.

Is there a way to add options to this rule to only look at scope of the current block, not child blocks?

### Rule name

noDuplicateProperties

### Playground link

n/a

### Expected result

Should not throw error

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
